homeassistant/config_entries.py

```diff
@@ -2397,6 +2397,7 @@ def async_update_reload_and_abort(
         data: Mapping[str, Any] | UndefinedType = UNDEFINED,
         options: Mapping[str, Any] | UndefinedType = UNDEFINED,
         reason: str = "reauth_successful",
+        reload_on_entry_update: bool = True,
```

Comment:
```suggestion
        always_reload: bool = True,
```

Maybe `always_reload` since it will still reload if the update changes something even if `reload_on_entry_update` is `False` which confused me when I reviewed the usage